circle.yml

```diff
@@ -19,8 +19,7 @@ dependencies:
 
 test:
   override:
-    - echo "Skipped...Running our uitests on every pull request breaks"
-    - echo "them irregularly and they're badly maintained at the moment."
+    - echo "Beep Boop"
```

Comment:
This doesn't revert anything though, it just changes the message we echo to the console..